.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '4.2'
+- '8'
```

Comment:
I recommend we also add simply `node` to this to make sure we are always testing against the latest stable release. https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions

Reply:
```yml
node_js:
    '8'
    'node'
```

Reply:
I agree we should do this, but we will need to adjust the config so we only deploy only once. So I think we should do this in a separate PR.

Reply:
Captured this in https://github.com/LLK/scratch-www/issues/1675